tools/gen_agency_cert.sh

```diff
@@ -98,6 +98,42 @@ parse_params()
     done
 }
 
+generate_cert_conf()
+{
+    local output=$1
+    cat << EOF > ${output} 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5344398)